package.json

```diff
@@ -19,7 +19,7 @@
     "dev": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot --progress --no-info",
     "lint": "npm run lint-ts && npm run lint-js && npm run lint-css",
     "lint-ts": "tslint './src/**/*.ts'",
-    "lint-js": "eslint '**/*.js' --ignore-path .gitignore",
+    "lint-js": "eslint '**/*.js'",
```

Comment:
One problem with this is that now we have the ignore defs duplicated in two different files: .gitignore and .eslintignore.

The original approach is more DRY so unless there's a compelling reason I tend to prefer it.